app/backend/dev-src/teet/test_setup.clj

```diff
@@ -114,10 +122,67 @@
                                                     :email (:user/email u)
                                                     :id (:user/id u)})}))}))
 
+
+;; Cadastral unit results for tests
+(def cadastral-unit-results
+  [{:KINNISTU "5477850",
+    :L_AADRESS "58 Aluste-Kergu tee",
+    :TUNNUS "12601:004:0003",
+    :link/external-id "2:12601:004:0003",
+    :link/type :cadastral-unit}
+   {:KINNISTU "5164950",
+    :L_AADRESS "58 Aluste-Kergu tee",
+    :TUNNUS "12601:001:0087",
+    :link/external-id "2:12601:001:0087",
+    :link/type :cadastral-unit}
+   {:KINNISTU "5453150",
+    :L_AADRESS "58 Aluste-Kergu tee"
+    :TUNNUS "12601:004:0004",
+    :link/external-id "2:12601:004:0004",
+    :link/type :cadastral-unit}
+   {:KINNISTU "3515350",
+    :L_AADRESS "Juurdepääsutee lõik 471",
+    :TUNNUS "12601:001:0093",
+    :link/external-id "2:12601:001:0093",
+    :link/type :cadastral-unit}
+   {:KINNISTU "3515550",
+    :L_AADRESS "Rapla-Pärnu raudtee 471",
+    :TUNNUS "12601:001:0050",
+    :link/external-id "2:12601:001:0050",
+    :link/type :cadastral-unit}
+   {:KINNISTU "54128140",
+    :L_AADRESS "Tartu maantee",
+    :TUNNUS "12601:001:0193",
+    :link/external-id "2:12601:001:0193",
+    :link/type :cadastral-unit}])
+
+
+(defn setup-mock-cadastral-unit-link-search [_req]
```

Comment:
Does this get called anywhere? WIP for now?

Reply:
It is used dynamically from the ring handler. Calling `cy.setup(foo, payload)` will invoke `setup-foo` function in the test setup namespace on the backend.